providers/default.rb

```diff
@@ -78,10 +79,14 @@ def update_rubygems
       end
       raise 'cannot find omnibus install' unless ::File.exist?(gem_bin)
       source = "--clear-sources --source #{new_resource.rubygems_url}"
-      shell_out!("#{gem_bin} update --system --no-rdoc --no-ri #{source}")
+      if Gem::Requirement.new(nodoc_rubygems_versions).satisfied_by?(rubygems_version)
+        shell_out!("#{gem_bin} update --system --no-document #{source}")
+      else
+        shell_out!("#{gem_bin} update --system --no-rdoc --no-ri #{source}")
+      end
```

Comment:
I guess we don't need this conditional here right now, but if we were ever to bump the target version up to >= 3.x then we would.